modules/react-components/src/components/grid/resource-grid/resource-grid.tsx

```diff
@@ -97,6 +97,27 @@ export const ResourceGrid: FunctionComponent<
             wrapperClassName
         );
 
+        if (isLoading) {
+            return (
+                <Card.Group style={ { marginBottom: "3rem" } }>
+                    {
+                        [ ...Array(4) ].map(() => {
+                            return (
+                                //TODO: Add style classes
+                                <>
```

Comment:
Redundant fragment. Is the TODO still applicable?
```suggestion
                                <>
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/4d9d7327a9f3509da0b1bc4492c8777dfdbec431